index.html

```diff
@@ -50,6 +52,194 @@ <h4 class="connect">LET'S CONNECT</h4>
+        <div class="cards-img"><img src="./img/snapshoot-d.png" alt="card-image"></div>
+        <h2 class="h2">Multi-Post<br>Stories</h2>
+
+        <ul class="top">
+          <li class="canopy">CANOPY</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">Back End Dev</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+
+        <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="projects">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+
+      </div>
+
+    </section>
+
+    <!-- About-Section -->
+    <section class="container-section">
+
+      <div class="intro2">
+        <h1>About<br>Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+
+        <h4 class="connect">LET'S CONNECT</h4>
+
+        <ul class="socials2">
+          <li><a href="#">
+              <img src="./img/github.svg" alt="Githud-Icon"></a>
+          </li>
+          <li><a href="#">
+              <img src="./img/linkedin-icon.svg" alt="LinkedIn-Icon"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/angellist.svg" alt="Angellist-Icon"></a>
+          </li>
+          <li><a href="#">
+              <img src="./img/twitter.svg" alt="Twitter-Icon"></a>
+          </li>
+
+          <li><a href="#">
+              <img src="./img/medium.svg" alt="Medium-Icon"></a>
+          </li>
+        </ul>
+
+        <div>
+          <button class="btn" type="submit">Get My Resume</button>
+        </div>
+
+        <ul>
+          <li class="langs">
+            <h2>Languages</h2>
+            <span class="chevron"><i class="fa-solid fa-chevron-down"></i></span>
+          </li>
+        </ul>
+
+        <ul class="work-list">
+          <li><img src="./img/jscript.png" alt="js-icon">
+          <h4 class="text">JavaScript</h4></li>
+          <li class="html"><img src="./img/html.png" alt="html-icon">
+          <h4 class="text">HTML</h4></li>
+          <li class="css"><img src="./img/css.png" alt="">
+          <h4 class="text">CSS</h4></li>
+        </ul>
+
+        <ul>
+          <li class="frame-skills">
+            <h2>Frameworks</h2>
+            <span class="chevron"><i class="fa-solid fa-chevron-right"></i></span>
+          </li>
+          <li class="frame-skills">
+            <h2>Skills</h2>
+            <span class="chevron"><i class="fa-solid fa-chevron-right"></i></span>
+          </li>
+        </ul>
```

Comment:
Kindly wrap `Languages`, `Frameworks` and `Skills` inside the same `ul` not two separate one contains `Languages` and one contains  `Frameworks` and `Skills`.